rest_framework/validators.py

```diff
@@ -211,7 +214,9 @@ def __call__(self, attrs):
         queryset = self.exclude_current_instance(attrs, queryset)
         if queryset.exists():
             message = self.message.format(date_field=self.date_field)
-            raise ValidationError({self.field: message})
+            raise ValidationError({
+                self.field: message,
+            })
```

Comment:
This the touchy part. 
I don't think we should be adding the code to  `ValidationError` that take a list or a dict as their first argument.
The code should only mean something directly attached to a single error. Not to a list or a dictionary, even if they only contain one error.